satpy/etc/composites/abi.yaml

```diff
@@ -824,3 +824,16 @@ composites:
           - name: C07
           - name: C13
     standard_name: blowing_snow
+
+  day_cloud_type:
+    description: >
+      Day Cloud Type, for GOES-R: NASA/NOAA
+    references:
+      Quick Guide: http://cimss.ssec.wisc.edu/goes/OCLOFactSheetPDFs/ABIQuickGuide_Day_Cloud_Type_RGB.pdf
+      EUMeTrain Guide: https://resources.eumetrain.org/data/7/736/navmenu.php?tab=3&page=1.0.0
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+      - name: C04
+      - name: C02
+      - name: C05
```

Comment:
these should normalised, I'm suggesting the pathlength correction.

Reply:
Was it decided that all channels that can be normalized should be normalized? The definition in the CIRA PDF does not explicitly mention it, but I assume that that version of this recipe does not have any corrections.

Reply:
Yes, solar channels should be normalised for rgbs.